pkg/domain/scan.go

```diff
@@ -20,19 +19,22 @@ type StatusHost struct {
 type ResultHost struct {
 	Host string `json:"id,omitempty"`
 }
+
 type Scan struct {
 	ID           string          `json:"id,omitempty"`
+	TenantID     string          `json:"tenant_id,omitempty"`
+	OperatorID   string          `json:"operator_id,omitempty"`
+	HostIDs      []int           `json:"host_ids,omitempty"`
 	HostsStatus  []StatusHost    `json:"hosts_status,omitempty"`
 	HostsResults []ResultHost    `json:"hosts_results,omitempty"`
 	Targets      []events.Target `json:"targets,omitempty"`
-	CreatedAt    time.Time       `json:"created_at"`
-	UpdatedAt    time.Time       `json:"updated_at"`
+	StartedAt    time.Time       `json:"started_at"`
+	EndedAt      time.Time       `json:"ended_at"`
 }
 
 func NewScan() *Scan {
 	return &Scan{
-		ID:        uuid.NewString(),
-		CreatedAt: time.Now().UTC(),
-		UpdatedAt: time.Now().UTC(),
+		StartedAt: time.Now().UTC(),
+		EndedAt:   time.Now().UTC(),
```

Comment:
It would be less confusing if we leave `EndedAt` as it's [zero-value](https://yourbasic.org/golang/default-zero-value/) on creation 
```suggestion
		return &Scan{
		    ID: uuid.NewString(),
		    StartedAt: time.Now().UTC(), // If you're posting a ScanStartedEvent when you create an instance, else it might not make too much sense
		    CreatedAt: time.Now().UTC(),
		    UpdatedAt: time.Now().UTC(),
```